tensorflow/lite/ios/ios.bzl

```diff
@@ -78,6 +79,59 @@ def tflite_ios_framework(
         ],
     )
 
+# iOS xcframework with symbol allowlist. Exported C++ symbols might cause symbol
+# collision with other libraries. List of symbols to allowlist can be
+# generated by running `nm -m -g FRAMEWORK_LIBRARY | grep _TfLite` for framework
```

Comment:
Could you help update the documentation here? This is for xcframework built with `apple_static_xcframework` rule?